packages/app/src/SearchPage.tsx

```diff
@@ -685,8 +692,11 @@ function SearchPage() {
       </Head>
       <SaveSearchModal
         show={showSaveSearchModal}
+        mode={saveSearchModalData.mode}
         onHide={() => setShowSaveSearchModal(false)}
         searchQuery={displayedSearchQuery}
+        searchName={selectedSavedSearch?.name ?? ''}
+        searchID={saveSearchModalData.searchID ?? ''}
```

Comment:
Can't this always be derived from `selectedSavedSearch?._id`?

Reply:
changes made